### PR TITLE
NXP-27826: renaming AI_Corpus to DatasetExport

### PR DESCRIPTION
**Pull request by andreinechaev**

### CODE REVIEW

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java [hunk 1]
```diff
@@ -66,17 +66,17 @@
     public static final String DATASET_TEMPLATE = "{\n" +
             "  \"entity-type\": \"document\",\n" +
             "  \"name\": \"%s\",\n" +
-            "  \"type\": \"AI_Corpus\",\n" +
+            "  \"type\": \"DatasetExport\",\n" +
             "  \"properties\": {\n" +
             "    \"dc:title\": \"%s\",\n" +
-            "    \"ai_corpus:documents_count\": %s,\n" +
```
Comment (covolution) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java]: This is the document on the cloud.  It needs to be "ai_corpus", don't change this.

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java [hunk 2]
```diff
@@ -19,40 +19,40 @@
 package org.nuxeo.ai.model;
 
 /**
- * Simple class to store constants related to AI Document Types: Ai_Model and Ai_Corpus
+ * Simple class to store constants related to AI Document Types: Ai_Model and DatasetExport
  */
 public class AiDocumentTypeConstants {
 
     // corpus type constants
-    public static final String CORPUS_TYPE = "AI_Corpus";
```
Comment (covolution) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java]: If this is used for cloud upload then it shouldn't be changed.

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java [hunk 3]
```diff
@@ -19,40 +19,40 @@
...
+    public static final String DATASET_EXPORT_TYPE = "DatasetExport";
 
-    public static final String CORPUS_SCHEMA = "ai_corpus";
```
Comment (covolution) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java]: If this is used for cloud upload then it shouldn't be changed.

File: nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
...
...6d4715ec5b","path":"/Nuxeo Jira/datasets/fb4a4505-90c6-4cc2-b1c6-e58c853fd148","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"4-0","isTrashed":false,"title":"T+D>Creator","lastModified":"2019-05-25T07:59:07.601Z","facets":[]},{"entity-type":"document","repository":"default","uid":"6c9b51a7-9cd7-449d-85e2-8e9a560e47b6","path":"/Nuxeo Jira/datasets/c1807187-c3d6-493d-85a5-fd7ddfb3ba73","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"3-0","isTrashed":false,"title":"Versions","lastModified":"2019-05-24T22:01:48.709Z","facets":[]},{"entity-type":"document","repository":"default","uid":"a3660985-5ece-41fd-9dfc-4f5e6e43975a","path":"/Nuxeo Jira/datasets/dd6e397b-9935-4b89-8202-cb32c4beaf54","type":"DatasetExport","state":"project","parentRef":"d45db34a-7d10-494d-b482-337c116fa224","isCheckedOut":true,"isVersion":false,"isProxy":false,"changeToken":"3-0","isTrashed":false,"title":"title+description+components","lastModified":"2019-05-23T15:54:15.994Z","facets":[]}]},
```
Comment (covolution) [inline nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json]: I think this should be `dataset_export`

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java (hunk 3 above)
Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java]: it shouldn't be. I created this schema.

File: nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json (hunk 4 above)
Comment (andreinechaev) [reply nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json]: no, it's a type not a schema

Comment (andreinechaev) [reply nuxeo-ai-jaxrs/src/test/resources/mappings/nuxeo_datasets.json]: we had AI_Corpus type that conforms to `ai_corpus` schema. Now we will have `DatasetExport` type that conforms to `dataset_export` schema

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java (hunk 1 above)
Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java]: ok

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java (hunk 2 above)
Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/model/AiDocumentTypeConstants.java]: don't see any usage.

**Pull request merged**